statistics.go

```diff
@@ -0,0 +1,60 @@
+package main
+
+import (
+	"io/ioutil"
+	"log"
+
+	"golang.org/x/net/context"
+	"golang.org/x/oauth2/google"
+	"github.com/Iwark/spreadsheet"
+	"regexp"
+	"strings"
+	"time"
+)
+
+type EmployeeAnniversary struct {
+	Name string
+	Date string
+	WorksYears int
+	Color string
+}
+
+func getWorkAnniversaries() [12][]EmployeeAnniversary {
```

Comment:
Why do you iterate through all 12 month if you only want to get information for one month?